domain-auth-methods.md

```diff
@@ -0,0 +1,118 @@
+
+# Domain Authorization For US Federal Government Public Trust
+
+The following summarizes the options available for Domain Authorization
+
+> Document content based on CAB Forum Baseline Requirements CP 1.4.5
+> https://cabforum.org/wp-content/uploads/CA-Browser-Forum-BR-1.4.5.pdf
+
+## Relevant CAB Forum Baseline Requirements Changes
+
+CAB Forum recently removed options that were previously acceptable. A summary of the options that were permitted prior to these changes can be found below in Appendix 1.
+
+| **Date** | **Section&nbsp;&nbsp;** | **Summary Description** |
+|:---------|:---------------- |:----------------------------------------------|
+| 2017-01-07 | 3.2.2.4 | Removal of validation methods 1,2,3,4,7,8, and 9. Method 11 added. |
+| 2017‐03‐01 |  3.2.2.4 | CAs MUST follow revised validation requirements in section 3.2.2.4 |
+| 2017‐04‐22 | 4.2.1 | Re‐use of validation information limited to 825 days |
+
+## Domain Authorization Methods
+
+1. [ Deleted ]
+2. [ Deleted ]
+3. [ Deleted ]
+4. [ Deleted ]
+
+5. **Domain Authorization Document** - Based on the attestation to the authority of the Applicant to request a certificate contained in a Domain Authorization Document
+	-	Domain Authorization Document must substantiate that the communication came from the Domain Contact
+	-	CA must verify that the Domain Authorization Document was either
+		-	Dated on or after the date of the domain validation request or
+		-	The WHOIS data has not materially changed since any previously provided Domain Authorization Document for the Domain Name Space
+
+6. **Agreed‐Upon Change to Website** - By confirming one of the following under “/.well-known/pki-validation” directory or other path registered with IANA for the purpose of Domain Validation, on the Authorization Domain Name that is accessible by the CA via HTTP/HTTPS over an Authorized Port
```

Comment:
process flow needed so we can look at each element and break it down by procedures across govt (and determine if there are blockers in govt wide policy or processes)